core/src/components/radio/test/label-placement/radio.e2e.ts

```diff
@@ -0,0 +1,110 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+/**
+ * By default ion-radio only takes up
+ * as much space as it needs. Justification is
+ * used for when the radio takes up the full
+ * line (such as in an ion-item). As a result,
+ * we set the width of the radio so we can
+ * see the justification results.
+ */
+test.describe('radio: label', () => {
```

Comment:
Could we have these radios `checked` by default? Since an unchecked radio isn't visible on `ios` mode, those screenshots aren't super useful for showing the positioning.